README.md

```diff
@@ -0,0 +1,95 @@
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
```

Comment:
:muscle: You've done a nice job here, @ceddddy , we just need to check a few details :wink: 
- [ ] Some important section are missing from your ReadMe and this is affecting its rating at the moment. Please go over this [template](https://github.com/microverseinc/readme-template) and add the missing sections/links. A descriptive ReadMe can potentially distinct you as a professional that you are. It makes your work more understandable and helpful to all types of users. :love_you_gesture: 
- _The only link/section that you are allowed to remove is the `FAQs` option, meaning that including other links will make your documentation more descriptive and understandable._
- You might also observe that the template is more desriptive :point_down: :
![image](https://user-images.githubusercontent.com/99715640/211677732-a1d0831d-638b-4786-aeb4-600f9bb11556.png)